swagger_parser/lib/src/parser/model/universal_component_class.dart

```diff
@@ -1,22 +1,25 @@
 part of 'universal_data_class.dart';
 
 /// Universal template for containing information about component
-@immutable
 final class UniversalComponentClass extends UniversalDataClass {
   /// Constructor for [UniversalComponentClass]
-  const UniversalComponentClass({
+  UniversalComponentClass({
```

Comment:
I had to remove the const constuctor as the `discriminatorValue` field needs to be updated after the initial UniversalComponentClass class is constructed.